README.txt

```diff
@@ -2,4 +2,4 @@
 unix: export PIPENV_VENV_IN_PROJECT=1
 windows: SET PIPENV_VENV_IN_PROJECT=1
 
-pipenv install --python 3.8 --dev
\ No newline at end of file
+pipenv install --python C:\Users\m.taghavi\AppData\Local\Programs\Python\Python38\python.exe --dev
```

Comment:
Always try to avoid reference to a local setup, use this:
```suggestion
pipenv install --python 3.8 --dev

Alternative:
pipenv install --python <FULL_PATH_TO_PYTHON\python.exe --dev
```